pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>${p.type}</packaging>
 
     <properties>
-        <models.version>7.0.3</models.version>
+        <models.version>6.1.5</models.version>
```

Comment:
Is this right?

Reply:
it is right, in Java we are still stuck with build `6.1` and remember that Java packaging does not support the different versions

Reply:
(I know, it is horrible)